lib/DDG/Meta/Helper.pm

```diff
@@ -64,6 +65,17 @@ Use booleans true and false to set options.
 
     $stash->add_symbol('&true', sub { 1 });
     $stash->add_symbol('&false', sub { 0 });
+
+=keyword rand_int
+
+generates random integer between 0 and supplied max integer (max defaults to 1).
+
+=cut
+
+	$stash->add_symbol('&rand_int', sub { 	
+		my $max = shift || 1;
```

Comment:
@nilnilnil @GuiltyDolphin Rationale for max default of `1` was there should be __a__ default and choosing any value above 1 seemed random. ;-)  The choice of `1` was for a coin-flip (true-false) random result. I couldn't decide on a larger default value; should it be 10, 100, etc?

As for not offering  a `$min` arg as well; the core `rand` just takes a `$max`, so it seemed consistent. It's also very easy to pass the `$max` value as the range between min/max, and then just add $min to value returned `rand_int`. 

There are other random functions that we might want to offer, such as selecting random item from list, or `rand_range` for passing a min/max pair. But looking at use of `rand` in existing Goodies, `rand_int` should cover all current usage.

Semi-related topic; I currently working on an issue with [Dice Goodie](https://github.com/duckduckgo/zeroclickinfo-goodies/issues/1231), and that module is now using `rand_int` helper function. Can I specify a dependency on DDG module/repo? Assuming the new helper function is accepted; how long before it makes it into production?